src/main.js

```diff
@@ -0,0 +1,42 @@
+function strComp (str1, str2) {
+  if (str1.localeCompare(str2) === -1 || str1.localeCompare(str2) === 0){
+    return true;
+  }
+}
+
+function checkAndPush(inputElement, liArray) {
+  const tempLiItem = document.createElement("li");
+  tempLiItem.textContent = inputElement.name;
+  for (j=0; j<liArray.length; j++){
+    if (strComp(inputElement.name.toUpperCase(), liArray[j].textContent.toUpperCase()))
+      break;
+  }
+  return j;
+}
+
+function renderInput(input, container){
+  const dirItem = [];
+  const fileItem = [];
+
+  const dirList = document.createElement("ul");
+  const fileList = document.createElement("ul");
+
+  var i, j;
+  for (i=0; i<input.length; i++) {
+    const tempItem = document.createElement("li");
+    tempItem.textContent = input[i].name;
+    if (input[i].type === "directory")
```

Comment:
You can simplify the code here in two ways:

1. Use [`filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) to filter directories and files. Example:<br /><br />
     ```javascript
     const files = allEntries.filter(entry => entry.type == 'file');
     ```
2. You can sort an array using the [`sort` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) and providing your own `sortFunction` argument, as described in the documentation.